src/components/ResetPassword/ResetPassword.jsx

```diff
@@ -0,0 +1,192 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Joi from 'joi-browser';
+import queryString from 'query-string';
+import resetPasswordInputSchema from '../../schemas/resetPassword';
+import Input from '../Input/Input';
+import { resetPasswordAction } from '../../actions/user/resetPassword';
+import './ResetPassword.scss';
+import Form from '../Form/Form';
+import Button from '../Button/Button';
+
+/**
+ *
+ *
+ * @export {component} jsx
+ * @class ResetPassword
+ * @extends {Component}
+ */
+export class ResetPassword extends Component {
+state = {
+  password: '',
+  confirmPassword: '',
+  passwordError: '',
+  confirmPasswordError: '',
+  token: '',
+  response: {}
+};
+
+/**
+ * @returns {function} mount component
+ *
+ * @memberof ResetPassword
+ */
+componentDidMount = () => {
+  const { props: { location: { search } } } = this;
```

Comment:
I really don't understand this `search`

Reply:
Because I append the reset password token to the url with a query string, search in location object is referencing the token.